starter-code/index.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Lab Bootsrtap Revera cloning</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/open-iconic/1.1.1/font/css/open-iconic-bootstrap.min.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta/css/bootstrap.min.css" integrity="sha384-/Y6pD6FV/Vv2HJnA6t+vslU6fwYXjCFtcEpHbNJ0lyAFsXTsjBbfaDjzALeQsN6M" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="container" id="top-header">
+      <div class="row">
+        <h1 class="col">Revera</h1>
+        <nav class="col-sm-8 mainmenu">
+          <ul class="nav">
+            <li "nav-item"><a class="nav-link active" href="#">Home</a></li>
+            <li "nav-item"><a class="nav-link" href="#">About</a></li>
+            <li "nav-item"><a class="nav-link" href="#">Sample page</a></li>
+            <li "nav-item"><a class="nav-link" href="#">Portfolio</a></li>
+            <li "nav-item"><a class="nav-link" href="#">Blog</a></li>
+            <li "nav-item"><a class="nav-link" href="#">Parent page</a></li>
+            <li "nav-item"><a class="nav-link" href="#">Contact</a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <div class="container-fluid" id="slidebox">
+      <div class="row">
+        <div id="carousel" class="carousel slide" data-ride="carousel">
+          <div class="carousel-inner">
+            <div class="carousel-item active">
+              <img class="d-block w-100" src="img/slider.jpg" alt="First slide">
+              <div id="overlay"></div>
+              <div class="carousel-caption d-none d-md-block">
+                <h2>Etiam mauris tortor, pharetra quis lobortis in, pharetra in diam</h2>
+                <p>Duis tempus leo vitae ipsum viverra, blandit condimentum sapien porttitor. Duis porttitor sed metus eget mollis. Curabitur bibendum imperdiet tortor, ut pulvinar purus elementum nec. Suspendisse at erat luctus; hendrerit sapien sed, consectetur erat. Maecenas dignissim suscipit orci at molestie. Aenean fringilla dolor vitae lacus lacinia eleifend. Suspendisse et libero nunc. Etiam mauris tortor, pharetra […]</p>
+                <button type="button" class="btn btn-light">READ MORE</button>
+              </div>
+            </div>
+            <!--<div class="carousel-item">
+              <img class="d-block w-100" src="..." alt="Second slide">
+            </div>
+            <div class="carousel-item">
+              <img class="d-block w-100" src="..." alt="Third slide">
+            </div>-->
+          </div>
+          <a class="carousel-control-prev" href="#carouselExampleControls" role="button" data-slide="prev">
```

Comment:
The reason the image switch isn't working is probably since it's looking for #carouselExampleControls instead of #carousel. Check out this example: 
https://www.w3schools.com/bootstrap/tryit.asp?filename=trybs_carousel&stacked=h

Reply:
changed it! It works :) Thank you!